src/io/flutter/view/MultiIconSimpleColoredComponent.java

```diff
@@ -117,7 +114,7 @@ public PositionedIcon(Icon icon, int index) {
 
   private int myMainTextLastIndex = -1;
 
-  private final TIntIntHashMap myFragmentPadding;
+  private final Map<Integer, Integer> myFragmentPadding;
```

Comment:
Note: it looks like this changed behavior - before this, querying for an entry that doesn’t exist would return `0`; now, it throws an NPE.